packages/storage/src/implementation/constants.ts

```diff
@@ -18,49 +18,50 @@
  * @fileoverview Constants used in the Firebase Storage library.
  */
 
+/**
+ * Domain name for firebase storage.
+ */
+export const DOMAIN_NAME: string = 'firebasestorage.googleapis.com';
+
 /**
  * Domain and scheme for API calls.
  */
-export const domainBase: string = 'https://firebasestorage.googleapis.com';
+export const API_DOMAIN: string = `https://${DOMAIN_NAME}`;
```

Comment:
Endpoint or something similar?

Reply:
On second thought, these are only used sparingly, so we could avoid the naming discussion by simply removing them.